app.js

```diff
@@ -1,15 +1,28 @@
 const express = require("express");
-const { getTopics, getArticles } = require('./controllers')
+const { getTopics, getArticles, getArticleById } = require("./controllers");
 
 const app = express();
+app.use(express.json());
 
-app.get('/api/topics', getTopics)
+app.get("/api/topics", getTopics);
 
 app.get("/api/articles", getArticles);
 
+app.get("/api/articles/:article_id", getArticleById);
+
+app.use((err, req, res, next) => {
+  if (err.code === '22P02') { 
+    res.status(400)
+    res.send({ msg: 'bad request' })
+  } else {
+    next(err)
+  }
+}) 
+
```

Comment:
Could we please add some more error handling middleware for when the ID is a number but not a valid one. I.e. ID 23891 would be a 404 error.